angular/core/components/sidebar/sidebar.haml

```diff
@@ -0,0 +1,47 @@
+/ %md_sidenav.md-sidenav-left{md-component-id: "left", md_is_locked_open: "$mdMedia('sm-l')",  md-whiteframe: 4}
```

Comment:
What is this file? Is it commented out? Why isn't it translated? Why multiple links to inbox? What happened to conditionally showing classes based on whether the page is selected or not?